backend/agent/llm.py

```
@@ -0,0 +1,36 @@
+from pydantic import BaseModel
```

Ruff (0.11.9)

`pydantic.BaseModel` imported but unused

Remove unused import: `pydantic.BaseModel`

(F401)